src/app/tokens.scss

```diff
@@ -139,6 +139,9 @@ $color-pink-20: #ff8ac5;
 $color-pink-10: #ffb4db;
 $color-pink-05: #ffdef0;
 
+// Gradient
+$gradient-blue: radial-gradient(100% 100% at 100% 0%, #B833E1 0%, #9059FF 37.1%, #5B6DF8 61.4%, #0090ED 100%);
```

Comment:
$color-violet-50: #b833e1
$color-purple-50: #9059ff
$color-blue-40: #0090ed

#5B6DF8 (I couldn't find an exact match from the Nebula/secondary colors above, but it's hard to find "close enough" between two hex colors).